spec/client_spec.rb

```diff
@@ -93,8 +94,12 @@
   end
 
   context 'delete' do
-    it 'returns hash on success with object_id' do
-      expect(client.delete_posts(object_id: 1)).to be_a(Hash)
+    it 'returns Takeout::Response on success' do
+      expect(client.delete_posts(object_id: 1)).to be_a(Takeout::Response)
+    end
+
+    it 'returns hash in its body on success with object_id' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.